packages/block-directory/src/components/downloadable-block-author-info/index.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * WordPress dependencies
+ */
+import { Fragment } from '@wordpress/element';
+import { __, _n, sprintf } from '@wordpress/i18n';
+
+function DownloadableBlockAuthorInfo( { author, authorBlockCount, authorBlockRating } ) {
+	return (
+		<Fragment>
+			<span className="block-directory-downloadable-block-author-info__content-author">
+				{ sprintf( __( 'Authored by %s' ), author ) }
+			</span>
+			<span className="block-directory-downloadable-block-author-info__content">
+				{ sprintf( _n( 'This author has %d block, with an average rating of %d.', 'This author has %d blocks, with an average rating of %d.', authorBlockCount, authorBlockRating ), authorBlockCount, authorBlockRating ) }
```

Comment:
Does an average rating of 0 imply that the author has no ratings? If so, should we exclude the second part of this sentence when the average rating is 0? It sounds al little awkward otherwise:

> This author has 1 block, with an average rating of 0.

Reply:
Let's think about this in a follow-up PR.